### DISCUSSION TITLE
Re-upload updated QSOs to Clublog

### DISCUSSION THREAD

**Discussion by DJLax5** [magicbug/Cloudlog#2243] [Q&A]
Hi everyone!

I've been using cloudlog now for a while and just added the clublog synchronization. 
On the first upload to clublog all LOTW/QSL statuses were synced. But if now new LOTW confirmations come in, the updated QSOs will not be updated on Clublog. 

I can pull from LOTW into Clublog just fine, but that would be a manual step again.. 
I was just curious if this is intended/fixable. 

73,
Fabian

**Comment by int2001** [comment]
QRZ.com had the same issue. This will be fixed in https://github.com/magicbug/Cloudlog/pull/2386 
Cloudlog also fixed in: https://github.com/magicbug/Cloudlog/pull/2387

Whenever a LotW-Confirmation is incoming, qso is marked to reupload at qrz.com and clublog.

Regarding (Paper-)QSLs - working on it
